app/scripts/views/sign_up.js

```diff
@@ -226,6 +226,11 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    onPasswordBlur: function () {
+      var password = this.getElementValue('.password');
+      this.checkPasswordStrength(password);
```

Comment:
We'll have to add able, either here on in the mixin, so that only a subset of users see the behavior. @shane-tomlinson to create experiment and integrate able.